src/smart-components/workflow/add-groups/add-workflow-wizard.js

```diff
@@ -6,37 +6,48 @@ import { Wizard } from '@patternfly/react-core';
 import { addNotification } from '@redhat-cloud-services/frontend-components-notifications';
 import { addWorkflow } from '../../../redux/actions/workflow-actions';
 import SummaryContent from './summary-content';
-import WorkflowInfoForm from './stage-information';
-import SetStages from './set-stages';
+import WorkflowInfoForm from './workflow-information';
+import SetGroups from './set-groups';
 
 const AddWorkflow = () => {
-  const [ formData, setValues ] = useState({});
+  const [ formData, setValues ] = useState({ wfGroups: []});
   const dispatch = useDispatch();
   const { push } = useHistory();
 
   const rbacGroups = useSelector(({ groupReducer: { groups }}) => groups);
   const [ isValid, setIsValid ] = useState(formData.name !== undefined && formData.name.length > 0);
+  const [ stepIdReached, setStepIdReached ] = useState(1);
 
   const handleChange = data => {
     setValues({ ...formData,  ...data });
   };
 
+  const onNext = ({ id }) => {
+    setStepIdReached(stepIdReached < id ? id : stepIdReached);
```

Comment:
you should use the callback version of setter to avoid using value which is out of sync
```jsx
setStepIdReached(prevStepIdReached =>  prevStepIdReached < id ? id : prevStepIdReached);
```